### DISCUSSION TITLE
How to use MeshObjectPlot with StructuredGrid

### DISCUSSION THREAD

**Discussion by Samuelopez-ansys** [ansys/ansys-tools-visualization-interface#169] [Q&A] [answered]
Hi community!

I would like to use MeshObjectPlot with a StructuredGrid PyVista object, is it possible? 

Thanks!

**Comment by AlejandroFernandezLuces** [comment, marked as answer]
Thanks for the interest in the library!

Currently, it is not supported. I've opened an issue for this feature, I'll get to implement it as soon as I can 🙂

**Reply by AlejandroFernandezLuces** [reply]
After completing https://github.com/ansys/ansys-tools-visualization-interface/pull/180, now `pv.StructuredGrid` is supported in release 0.5.0 🚀 

You can check this example in the picker to see a simple example of how MeshObject would work: https://visualization-interface.tools.docs.pyansys.com/version/stable/examples/00-basic-pyvista-examples/picker.html#using-structuredgrid-mesh

**Reply by Samuelopez-ansys** [reply]
Wow! That was fast! Thank you, I will test it :)
